packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -314,9 +327,30 @@ public function import_term( $data ) {
 
 				// Wipe the parent for now
 				$data['parent'] = 0;
+			}*/
+			$parent_term = term_exists( $parent, $data['taxonomy'] );
+
+			if ( $parent_term ) {
+				$data['parent'] = $parent_term['term_id'];
+			} else {
+				// It can happens that the parent term is not imported yet in manually created WXR files.
+				$parent_term = wp_insert_term( $parent, $data['taxonomy'] );
```

Comment:
What happens if the parent's parent is also missing? In other words, imagine we have a hierarchy of 10 nested parents and we're to process a term at the very bottom of that hierarchy?

Here's a visual representation:

```
Root Parent
    │
    ├── Parent Level 1
    │       │
    │       ├── Parent Level 2
    │       │       │
    │       │       ├── Parent Level 3
    │       │       │       │
    │       │       │       ├── Parent Level 4
    │       │       │       │       │
    │       │       │       │       ├── Parent Level 5
    │       │       │       │       │       │
    │       │       │       │       │       ├── Parent Level 6
    │       │       │       │       │       │       │
    │       │       │       │       │       │       ├── Parent Level 7
    │       │       │       │       │       │       │       │
    │       │       │       │       │       │       │       ├── Parent Level 8
    │       │       │       │       │       │       │       │       │
    │       │       │       │       │       │       │       │       ├── Parent Level 9
    │       │       │       │       │       │       │       │       │       │
    │       │       │       │       │       │       │       │       │       └── Term
    │       │       │       │       │       │       │       │       │           (Processing here)
    │       │       │       │       │       │       │       │       │
    │       │       │       │       │       │       │       │       └── ...
    │       │       │       │       │       │       │       └── ...
    │       │       │       │       │       │       └── ...
    │       │       │       │       │       └── ...
    │       │       │       │       └── ...
    │       │       │       └── ...
    │       │       └── ...
    │       └── ...
    └── ...
```

Reply:
Also, why would a parent be missing if we've already sorted everything topologically?